### PR TITLE
test changes made

### PR DESCRIPTION
**Pull request by 1lyvianis**
edited a few rows

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # git-tutorials
 
-a tutorial for demonstrating git usage and lecturing.
+a tutorial for demonstrating git usage and lecturing
+changes!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```
Comment (hsinatfootprintai) [inline README.md]: why do you add this?

**Review by hsinatfootprintai — CHANGES_REQUESTED** [review]
please fix it to be meaningful

**Comment by 1lyvianis** [comment]
> please fix it to be meaningful

discarded previous codes to comply with the latest version

File: README.md (hunk 1 above)
Comment (1lyvianis) [reply README.md]: done

**Review by hsinatfootprintai — APPROVED** [review]

**Pull request merged**